dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -814,7 +814,7 @@ JSON_FACTORY, new GenericUrl(orcidUrl + "oauth/token"),
             orcid = tokenResponse.get("orcid").toString();
             scope = tokenResponse.getScope();
             Instant instant = Instant.now();
-            instant.plusSeconds(tokenResponse.getExpiresInSeconds());
```

Comment:
Yikes, this one was bad. A code analysis tool found this? I couldn't tell which one from the screenshot.

Reply:
Yeah, sonar cloud